frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -952,7 +952,9 @@ function FeatureFlagRollout({ readOnly }: { readOnly?: boolean }): JSX.Element {
                             </div>
                             <div className="col-span-4 flex items-center gap-1">
                                 <span>Rollout</span>
-                                <LemonButton onClick={distributeVariantsEqually}>(Redistribute)</LemonButton>
+                                <LemonButton onClick={distributeVariantsEqually} size="xsmall">
+                                    (Redistribute)
+                                </LemonButton>
```

Comment:
I didn't like how big this button was alongside the `Rollout` header, they were too close in size and that made it feel like the button was a non-obvious CTA for an action.  This change differentiates the two elements more distinctly, which makes the fact that `Redistribute` is a button more obvious

Before
(not selected)
<img width="303" alt="image" src="https://github.com/user-attachments/assets/ae5f76f6-a55c-4a11-abc3-472a005f8632">
(selected)
<img width="335" alt="image" src="https://github.com/user-attachments/assets/7ecdf329-170d-4ed2-b6ec-3d918b82449a">


After
(not selected)
<img width="345" alt="image" src="https://github.com/user-attachments/assets/3b87066d-e6a6-4ba3-8fb4-2a9aa6d4c2e9">

(selected)
<img width="363" alt="image" src="https://github.com/user-attachments/assets/9182685a-5363-498c-9d51-a0c781294f82">



That said, open to feedback on not doing this.  cc @jurajmajerik

Reply:
Hm, when not selected/hovered it looks weird to have the different font size. I'd prob use `type="secondary"`, remove the parentheses, no padding, and align it over on the right. 

<img width="1156" alt="Arc 2024-10-14 19 51 58" src="https://github.com/user-attachments/assets/b2981d03-a639-4f6c-974e-6cf6f09643f2">

I still don't love this mostly because we're fairly constrained on width here, I think an icon with tooltip might be better, but I don't see one that makes a lot of sense. @corywatilo might be inspired, maybe something like a 100 with an arrow or something.